examples/morphology.ipynb

```diff
@@ -134,7 +134,6 @@
     "    strain=strain,\n",
     "    brain_region=brain_region,\n",
     "    location=brain_location,\n",
-    "    legacy_id=\"temp\",\n",
```

Comment:
@GianlucaFicarelli it seems that legacy_id is mandatory when registering a morphology. Could we perhaps change that?

Reply:
Yes I see that it's mandatory but it can be None. I'll add a default value to None.

Reply:
Yes I see that it's mandatory but it can be None. I'll add a default value to None.

Reply:
That will do for now; set it to None.